addon/model-data.js

```diff
@@ -517,6 +540,14 @@ export default class M3ModelData {
       // destroy
       delete this._childModelDatas[key];
     }
+    if (this._projections) {
+      // TODO Add a test for this destruction
+      for (let i = 0; i < this._projections.length; i++) {
```

Comment:
We can start from `i = 1` here, as we know that the base is always the very first element. This will save us the `if` statement below.